views/components/Graphs/BarGraph.jsx

```diff
@@ -23,7 +23,8 @@ const BarGraph = ({ graph }) => {
         <VictoryVoronoiContainer
           labels={({ datum: { study, studyTarget, count, valueLabel } }) => {
             const showToolTip = study && count
-            return valueLabel !== 'N/A' && showToolTip
+            const notTotals = study !== 'Totals'
+            return valueLabel !== 'N/A' && showToolTip && notTotals
```

Comment:
I wonder if we want to update the `showToolTip` logic with this change. What do you think? Also, do you think we could leverage the same constant in both places for the totals study?

```js
const showToolTip = study && count && study !== 'Totals'
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/137/commits/63f1716f2a06552e8dee7d012e9a7733b206fbaa